### ISSUE TITLE
Anchor tag attribute target="{my value}" seems to be ignored

### ISSUE THREAD

**Issue by netaisllc** [layerJS/layerJS#75]
HTML anchor tags with a truthy value on the **_target_** attribute should open in a new browser window.  It seems using _layerJS_ defeats this, instead loading the content of the href into the current browser window.

**Steps To Repro:  (Based on a simple wiki example)**

1. SETUP:: Eliminate layerJS by commenting out cdn links.

```
<html>
<head>
    <!--    
    <script src="http://cdn.layerjs.org/libs/layerjs/layerjs-0.5.2.min.js"></script>
    <link href="http://cdn.layerjs.org/libs/layerjs/layerjs-0.5.2.css" type="text/css" rel="stylesheet" />
    -->
</head>
<body>
    <div lj-type="stage" style="width:100%; height:500px">
        <div lj-type="layer" lj-default-frame="frame1">
            <div lj-type="frame" lj-name="frame1">
                <a href="https://google.com" target="_new">Google</a>
            </div>
            <div lj-type="frame" lj-name="frame2">
                <a href="https://linkedin.com" target="_new">LinkedIn</a>
            </div>
        </div>
  </div>
  <a href="#frame1">Frame 1</a>
  <a href="#frame2">Frame 2</a>
</body>
</html>
```

2. RESULT: external links open in new browser windows (tabs).

3. ELABORATION: Uncomment the LayerJS links.

4. NEW RESULT: external links open in the current browser window (tab)

I'm guessing the libraries Anchor Tag handler is not considering the target attribute...  

IMHO it should. In particular as you guys are championing using 'only markup'.

Thanks!

**Comment by thomashandorf** [comment]
Thanks for reporting! Makes sense. We should should add in router.js -> $.addDelegtedListener(function()) a check to the if clause: if a target is given and the target is not "_self", navigate shouldn't be called and default should not be prevented

**Comment by netaisllc** [comment]
Such as...
```
/*
 * Issue #75 :: Anchor tag attribute target="{my value}" seems to be ignored
 * as at 9-May-2018
 * 
 * Spec :: We should should add in router.js -> $.addDelegtedListener(function()) a check to the if clause:
 *    if 
 *      a target is given AND the target is not "_self", 
 *    then
 *      navigate shouldn't be called AND default should not be prevented
 **/

// register link listener
$.addDelegtedListener(this.rootElement, 'click', 'a:not([data-lj-nolink=\'true\']):not([lj-nolink=\'true\'])', function(event) {
    //delete that.ignoreUrl;
    
    var _navigationHandler = function(result) {
      if (!result) {
        setTimeout(function() { // why do we have to get at the end of the queue?
            that.ignoreUrl = $.getAbsoluteUrl(href);
            window.location.href = href;
        }, 1);
      }
    }

    if (this.href !== '' && !this.href.startsWith('javascript:')) { // jshint ignore:line
      // get the explicitly given href (that is not extended) to see the intention of the user
      var href = this.getAttribute('href');

      // check for target attr
      var tget = this.getAttribute('target');
      // if target is != '_self', honor it as user intention
      if( tget && tget !== '_self' ) {
        return
      }
      else {

        event.preventDefault(); // prevent default action, i.e. going to link target
        // do not stop propagation; other libraries may listen to link clicks
        that.navigate(href, $.findParentViewOfType(this, 'layer')).then( _navigationHandler );
      }    
    }
});
```

Would you like a PR?

**Comment by thomashandorf** [comment]
@netaisllc yes please, submit a PR (:

**Comment by thomashandorf** [comment]
fixed in 4533a9dba900637498098af82650e8e42c832fac
